src/help/zaphelp/contents/ui/dialogs/options/view.html

```diff
@@ -84,6 +84,11 @@ <H3>Font Size</H3>
 The 'Font Example' field will show you how large the default text will appear.<br>
 This setting will only take effect when ZAP is restarted.
 
+<H3>Look and Feel</H3>
+Allows to choose the look and feel of ZAP. If empty it's used a default, <code>Nimbus</code> in Windows and Linux, the native
+system look and feel in macOS.<br />
```

Comment:
How about:
> If empty the default is used, ...

Should macOS have a leading cap? (Not sure if there's a standard for that)

Reply:
Yeah, will change that. I don't think so, it's now written that way: https://www.apple.com/macos/

Reply:
Changed.